agent/src/main/java/com/code_intelligence/jazzer/static_analysis/SootJazzerCFGIdMapper.kt

```diff
@@ -87,7 +87,10 @@ fun <E> DefaultDirectedGraph<Unit, E>.simplify() {
         }
 }
 
-object SootJazzerCFGIdMapper {
+class SootJazzerCFGIdMapper {
+    val outputGraph = DefaultDirectedGraph<Unit, DefaultEdge>(DefaultEdge::class.java)
+    val visited = ArrayList<Unit>()
```

Comment:
Based on the usage below, this should probably be a set (otherwise, due to lookups taking linear time, the algorithm below could become quadratic in the number of visited units):
```suggestion
    val visited = mutableSetOf<Unit>()
```